README.md

```diff
@@ -142,6 +115,30 @@ LOW: 0
 
 The policy report is just one report type available in Curio. Additional options for using and configuring the `scan` command can be found in the [scan documentation](https://curio.sh/reference/commands/#scan). For additional guides and usage tips, [view the docs](https://curio.sh).
 
+## :gear: Additional installation options
+
+### Configurable install script
+
+The default installation script works well for most use cases, but if you need more control you can customize the options by passing additional parameters.
+
+- `-b`: sets the installation directory (defaults to `./bin`)
+- `-d`: enables debug logging
+- `[tag]`: specifies a tag release (defaults to the latest release)
```

Comment:
@cfabianski I'm not sure on the specifics for customizing the install script but I've included the more obvious flags. Not sure about the `--` that is used in the example though?

Reply:
That’s correct (including the —)